File: sorting-algos/bubble_sort/bubble-sort.cpp
#include<iostream.h>
 void main()
{
  int a[50],i,j,temp,n;
  cout<<"enter the elements";
  for(i=0;i<n;i++)
   {
    cin>>a[i];
    }
  for(i=0;i<n;i++)
  {
   for(j=0;j<(n-1);j++)
   {
    if(a[j>a[j+1])
    { 
     temp=a[j];
     a[j]=a[j+1];
     a[j+1]=temp;
     }
   }
  }
  cout<<"Elements after sorting";
  for(i=0;i<n;i++)
  cout<<a[i];
 }
     

File: sorting-algos/bubble_sort/bubble_sort.py
#**********BUBBLE SORT***************
# enter n value and enter that many number in your list
# in bubble sort adjacent values are comapared and swapped
# Input: Value of n, List Elements
# Output: Unsorted list, Sorted List

l = []
n = int(input("Enter no. of elements in list (n): "))
for i in range(n):
    l.append(int(input()))

print("Unsorted list: ")
print(l)

for i in range(n):
    for j in range(0,i+1):
        if l[j]>l[i]:
            temp = l[j]
            l[j] = l[i]
            l[i] = temp

print("Sorted list: ")
print(l)            